packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -108,37 +108,15 @@
     </app-template>
     <groups-modal
       v-if="addToGroupsModalIsOpen"
-      :title="$gettext('Add to groups')"
-      :message="
-        $ngettext(
-          'Add selected user %{user} to groups',
-          'Add %{userCount} selected users to groups ',
-          selectedUsers.length,
-          {
-            user: selectedUsers[0].displayName,
-            userCount: selectedUsers.length.toString()
-          }
-        )
-      "
+      :title="addToGroupsModalTitle"
       :groups="groups"
```

Comment:
Where has the message gone? 
The prop is marked required in the component

Reply:
Please see the screenshots in the linked pr. 
I purged the required prop